custom-tests.yaml

```diff
@@ -298,6 +298,11 @@ api:
       };
       var instance = canvas.getContext('2d');
     __additional:
+      drawFocusIfNeeded.path_parameter: >-
+        var path = new Path2D();
+        var el = document.createElement('p');
+        instance.drawFocusIfNeeded(path, el);
```

Comment:
If the detection is based on this throwing an exception, can you do an explicit try-catch?